lte/gateway/c/core/oai/test/spgw_task/test_spgw_procedures_with_injected_state.cpp

```diff
@@ -124,10 +124,16 @@ class SPGWAppInjectedStateProcedureTest : public ::testing::Test {
     spgw_app_init(&spgw_config, mme_config.use_stateless);
 
     // add injection of state loaded in SPGW state manager
+#ifdef CMAKE_BUILD  // if CMAKE is used the absolute path is needed
     std::string magma_root = std::getenv("MAGMA_ROOT");
     name_of_ue_samples = load_file_into_vector_of_line_content(
         magma_root + "/" + DEFAULT_SPGW_CONTEXT_DATA_PATH,
         magma_root + "/" + DEFAULT_SPGW_CONTEXT_DATA_PATH + "data_list.txt");
+#else  // if bazel is used the relative path is used
+    std::string path = DEFAULT_SPGW_CONTEXT_DATA_PATH;
+    name_of_ue_samples =
+        load_file_into_vector_of_line_content(path, path + "data_list.txt");
```

Comment:
Can we put this function commonly accessible for both builds and set the path variable for individual build.

Reply:
Changes pushed.